.github/workflows/ci-composer-version.yml

```
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Bypass the failing test...
```suggestion
      - uses: actions/checkout@v3
```